styles/netscape/catppuccin.user.css

```diff
@@ -8,13 +8,11 @@
 @description Soothing pastel theme for Netscape ISP
 @author Catppuccin
 @license MIT
-
```

Comment:
This whitespace is in the template, and is required for *aesthetics* and consistency.